### ISSUE TITLE
Support for swagger ui 3

### ISSUE THREAD

**Issue by iamleniac** [rdidyk/falcon-swagger-ui#2]
Any chance to do this?

**Comment by volcan01010** [comment]
I would be willing to contribute to this if you could give me some help.  I tried replacing the `dist` folder with a more recent version from [https://github.com/swagger-api/swagger-ui.git](https://github.com/swagger-api/swagger-ui.git).  The endpoint is created, but the resulting page is not correct.  I suppose that the changes from v2.x to v3.0 were significant.

**Comment by rdidyk** [comment]
Hi, @volcan01010 
Thank you very much for the offer!
The main thing to do is adapt the template: ` falcon_swagger_ui/templates/index.html` for the new version of swagger-ui.
It is a pity that now I do not have enough free time to migrate the library to a new version.

**Comment by volcan01010** [comment]
Thanks.  And thanks for creating the library in the first place.

I have managed to update the template to substitute in the `base_url` and `api_url`.  This gives me a working /swagger endpoint with the latest Swagger UI code.

https://github.com/volcan01010/falcon-swagger-ui/tree/update_swagger_ui

However, the latest Swagger UI has moved the appName, realm, clientSecret and clientId definitions out of the `index.html`.  I'm trying to see how to pass those through.  Any help would be gratefully received.  I don't want to break the OAuth stuff.

**Comment by volcan01010** [comment]
I have created a pull request: https://github.com/rdidyk/falcon-swagger-ui/pull/4

Running the example app (I added instructions to README) creates a working `/swagger` endpoint.  The config is printed to the console.  I have added the OAuth configuration code from here (https://github.com/swagger-api/swagger-ui/blob/743eb8c9c73ec77dc5dba8b101dd1b7ff1a3e1ff/docs/usage/oauth2.md) but I don't know how to test it.
